media-libs/vidstab/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>maintainer-wanted@gentoo.org</email>
```

Comment:
you dhpuld add yourself first if you are going to maintain it yourself first, and then add proxy-maintainers project last

Reply:
I am not 100% sure I did it right, at least repoman does not complain.
BTW my Bugzilla account is different that my GitHub :-|